crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased] - ReleaseDate
 
+- Fix [#4424](https://github.com/matrix-org/matrix-rust-sdk/issues/4424) Failed storage upgrade for
+  "PreviouslyVerifiedButNoLonger"
```

Comment:
That's pretty terse, can we expand on what and how this affects users. Also all the other entries link a PR or commit at the end, could we stick to that convention.

Reply:
Updated the description, rebased against `release-for-crypto-wasm-11` and retargeted this PR. Will open a new PR for the merge into main.

Reply:
New PR into main is https://github.com/matrix-org/matrix-rust-sdk/pull/4434